MapboxSearch/sdk/src/test/java/com/mapbox/search/SearchEngineTest.kt

```diff
@@ -773,6 +775,174 @@ internal class SearchEngineTest {
         }
     }
 
+    @TestFactory
```

Comment:
There's also `SearchEngineIntegrationTest`. This might be helpful to add few more integration test to see how search native handles manually created `SuggestAction` and `com.mapbox.search.internal.bindgen.SearchResult`,